### PR TITLE
Terms of service

### PR DESCRIPTION
**Pull request by arnonrdp**

### CODE REVIEW

File: src/pages/TermsOfService.vue [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      <b>6. Privacy and Data</b>
+      <p>
+        Celebrity Fanalyzer is committed to user privacy. We do not collect or store personal information beyond what is necessary for
+        account creation and participation. We will work towards implementing an anonymizing HTTP reverse proxy called "Layer 8" to further
+        protect user identity.
+      </p>
+
+      <b>7. Future Developments</b>
+      <p>
+        Celebrity Fanalyzer is a stepping stone to the development of "Globe & Citizen," an anthroscope news platform. Your use of Celebrity
```
Comment (stravid87) [inline src/pages/TermsOfService.vue:58]: ``
```suggestion
        Celebrity Fanalyzer is a stepping stone in the development of "Globe & Citizen," a Web3.0 enabled news platform. Your use of Celebrity
```

File: src/pages/TermsOfService.vue [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      </p>
+
+      <b>11. Changes to Terms</b>
+      <p>
+        Celebrity Fanalyzer may update these Terms of Service from time to time. Continued use of the platform after such changes implies
+        your acceptance of the updated terms.
+      </p>
+
+      <b>12. Governing Law</b>
+      <p>These Terms of Service shall be governed by and construed in accordance with the laws of [Insert Jurisdiction].</p>
```
Comment (stravid87) [inline src/pages/TermsOfService.vue:87]:
```suggestion
      <p>These Terms of Service shall be governed by and construed in accordance with those laws which are applicable.</p>
```

**Review by stravid87 — APPROVED** [review]
Great terms of service. I made only minor edits. 
I also like how you added the two icons. Very slick.

**Comment by arnonrdp** [comment]
> Great terms of service. I made only minor edits. 
> I also like how you added the two icons. Very slick.

Great!
Thanks, Captain.

**Pull request merged**